CactusTutorial.ipynb

```diff
@@ -535,11 +545,117 @@
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `submit` command submitted the simulation to the queueing system to run in the background asking for a maximum runtime of 20 minutes. On your laptop it will start right away, on a cluster the queuing system will wait until a sufficient number of nodes is able to start your simulation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can check the status of the simulation with the command below. You can run this command repeatedly until the job shows <pre>[ACTIVE (FINISHED)...</pre> as it's state. Prior to that, it may show up as QUEUED or RUNNING."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "bash: line 1: simfactory/bin/sim: No such file or directory\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "simfactory/bin/sim list-simulations tov)ET"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " To watch its output use the `show-output` command of simfactory. **Interrupt the kernel** (or press `CTRL-C` if copying & pasting these commands to a terminal) once you wish to stop watching or once the simulation has ended."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# watch log output, following along as new output is produced\n",
+    "./simfactory/bin/sim show-output --follow tov_ET"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can loeave out the `--follow` option if you would like to see all output up to this point."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Managing submitted simulations\n",
+    "\n",
+    "Since the `submit` command was used to start the  simulation, it is running in the background and you have to use simfactory commands to interact with it. The next two cells show how to list simulations and how to stop them."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "simfactory/bin/sim list-simulations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "simfactory/bin/sim stop tov_ET"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "after this the simulation changes into the \"FINISHED\" state indicating it is no longer running."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "simfactory/bin/sim list-simulations"
```

Comment:
So we do a list / a stop / and a list again. Presumably, we don't really want them to stop the simulation, we just want them to know how they can do it. Right? If so, we don't need two list simulations.

Reply:
The cell does not exit b/c the `--follow` option make `show-output` never exit. There is bolded text above that reads "Interrupt the kernel (or press CTRL-C if copying & pasting these commands to a terminal) once you wish to stop watching or once the simulation has ended.". I have added more bolded text just above the next cell (which will seem to hang) stating "Remember that you have to interrupt the kernel to stop show-output and be able to execute the cells below." as I was also caught be not realizing one has to interrupt even after the simulation finished.